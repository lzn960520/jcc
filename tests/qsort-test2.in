10
1 47 29 1498 487 126 410 71 1247 123
