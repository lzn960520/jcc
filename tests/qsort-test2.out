1 29 47 71 123 126 410 487 1247 1498
